src/wily/commands/graph.py

```diff
@@ -75,6 +77,7 @@ def graph(
         z_axis = resolve_metric(metrics[1])
         z_operator, z_key = metric_parts(metrics[1])
     for path in paths:
+        path = Path(path)
```

Comment:
This code was already pretty messy. The variable `path` has too many meanings. It's a parameter to the function, it's a local assignment in this loop, and then it's overridden with a Path constructor. 

Also, if you look at all the lines after 80 where `path` is used, it's cast back into a `str`.

Can you change this to something like `current_path = str(Path(path_))` and change the loop to `for path_ in paths`. Just to separate the variables.

Then you can remove the `str()` calls from lines 88, 97 and 109